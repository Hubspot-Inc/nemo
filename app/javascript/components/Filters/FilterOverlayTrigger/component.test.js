import React from 'react';
import { shallow } from 'enzyme';

import Component from './component';

const defaultProps = {
  id: 'id',
  title: 'title',
  overlay: 'overlay',
  hints: ['hint'],
};

it('renders as expected', () => {
  const wrapper = shallow(<Component {...defaultProps} />);
  expect(wrapper).toMatchSnapshot();
});
