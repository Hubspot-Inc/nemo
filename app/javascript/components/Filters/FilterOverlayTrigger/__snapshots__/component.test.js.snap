// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders as expected 1`] = `
<OverlayTrigger
  containerPadding={25}
  id="id"
  overlay="overlay"
  placement="bottom"
  rootClose={true}
  trigger="click"
>
  <Button
    id="id"
    variant="secondary"
  >
    title (hint)
  </Button>
</OverlayTrigger>
`;
